Dockerfile

```diff
@@ -5,3 +5,7 @@ LABEL authors="Peter J Bailey, Alexander Peltzer, Olga Botvinnik" \
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 ENV PATH /opt/conda/envs/nf-core-scrnaseq-1.0dev/bin:$PATH
+
+# Alternatively, get STAR source using git
```

Comment:
@olgabot please not - its hardly reproducible to do this, so instead we should update the STAR recipe (if required) on bioconda, can do that if you don't want to.